src/components/Navbar.tsx

```diff
@@ -7,11 +12,41 @@ import {
   userLogoutAction,
 } from 'hooks';
 import { Link } from 'react-router-dom';
+import {
+  lightTheme,
+  setTheme,
+} from 'tui__theme/index.js';
 
 export const Navbar = () => {
   const { loginUser, dispatchLoginUser, isAuth } = useLogin();
+  const [globalThemeDark, setGlobalThemeDark] = useState(true);
+  useEffect(() => {
+    let theme = localStorage.getItem("lg2__theme");
+    if (theme === "dark") {
+      setGlobalThemeDark(true);
+      setTheme();
+    } else if (theme === "light") {
+      setGlobalThemeDark(false);
+      setTheme({ theme: lightTheme });
+    } else {
+      setGlobalThemeDark(true);
+      setTheme();
+      localStorage.setItem("lg2__theme", "dark");
+    }
+  }, []);
+  const toggleTheme = () => {
+    if (globalThemeDark) {
+      setTheme({ theme: lightTheme });
+      localStorage.setItem("lg2__theme", "light");
+      setGlobalThemeDark(false);
+    } else {
+      setTheme();
+      localStorage.setItem("lg2__theme", "dark");
+      setGlobalThemeDark(true);
```

Comment:
This step can be abstracted as setGlobalThemeDark(! globalThemeDark)

Reply:
No, I do not intend to toggle it here.